CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+### NGI-RNAseq
```

Comment:
Hmm, yeah that would be nice. I looked at MultiQC_NGI for a reference though:
https://raw.githubusercontent.com/ewels/MultiQC_NGI/master/CHANGELOG.md
I guess the main MultiQC repo would have been a better inspiration...